cli/cli.go

```diff
@@ -66,6 +66,8 @@ type Config struct {
 	IP                string
 	Remote            string
 	Label             string
+	Responses         string
+	Path              string
```

Comment:
I guess Path is not really necessary here, as we right now hard-code the endpoints. And yes we should later refactor serve and ocspserve with proper routing treatment.